### DISCUSSION TITLE
Can't Verify Student ID

### DISCUSSION THREAD

**Discussion by Radrahil** [community/community#61072] [GitHub Education]
### Hi everyone,

I do not have a school mail that can receive e-mails (no mailbox). I have uploaded the Student ID Fifteen Times. It hasn't worked. I have also not received a reason as to why it did not work.

**Comment by nonvenomous** [comment]
Same issue here.
There are many people with this issue, basically every 4th discussion in the github education category is about this.
A big thread here https://github.com/orgs/community/discussions/59360

I would delete the picture or at least censor the picture of your student ID.

**Comment by ghost** [comment]
Would you mind elaborating on "hasn't worked" because we really cannot understand what happened if what happened was that it just didn't work.

**Comment by samus-aran** [comment]
Hi @Radrahil 

We're sorry you're experiencing this. Can we please ask you to try now and re-apply. We've raised this experience with our internal team but we hope this helps you proceed to re-verify your academic status. 👍
